lib/metadata_json_deps/rake_tasks.rb

```diff
@@ -1,17 +0,0 @@
-require 'metadata_json_deps'
```

Comment:
Where has this been moved to?

Reply:
The rake tasks are not used anymore. There is an executable in `bin`.